sdk/communication/azure-communication-callautomation/azure/communication/callautomation/__init__.py

```diff
@@ -120,5 +118,12 @@ def __getattr__(name):
         )
         from ._models import ServerCallLocator
         return ServerCallLocator
-
+    if name == 'MicrosoftBotProperties':
```

Comment:
Looks like `MicrosoftBotProperties` was never added to the API in the first place - so this probably isn't needed.